### DISCUSSION TITLE
invisible mouse

### DISCUSSION THREAD

**Discussion by ModeratelyModular** [Snoothy/UCR#317] [General]
Hi, 

Came here to use a USB-footswitch (FS) as a sustain pedal in Ableton. FS works like a mouse, is recognized by W10 as a HID. However UCR only detects MIDI-keyboard as device. I read UCR would work with 'any input' and mice are explicitly mentioned. But it doesn't recognize one when there are even two there ;-)  Nevertheless, during binding (button to button) the binding button lights up, so it receives signal.

I do like the interface. 

Kind regards

**Comment by Whosdeez** [comment]
For anyone reading this in the future, you can use this plugin https://github.com/snoothy/ucr/wiki/Core_Interception to capture keyboards and mice as inputs in UCR.
